src/loc/VoidLocButton.test.tsx

```diff
@@ -54,14 +54,12 @@ describe("VoidLocButton", () => {
         await waitFor(() => expect(dialog!).not.toBeVisible());
     });
 
-    it("does not void LOC on cancel", async () => {
-        let called = false;
+    it("disappears on cancel", async () => {
         const dialog = await renderAndOpenDialog();
 
         await clickByName("Cancel");
 
         await waitFor(() => expect(dialog!).not.toBeVisible());
-        expect(called).toBe(false);
```

Comment:
This expect should remain.

Reply:
No, because it only gives the illusion that it's tested against actual call, which is not the case. In this test, `called` was just a local var used nowhere else.